src/components/cartCard/cart-card.jsx

```diff
@@ -0,0 +1,117 @@
+import { React, Fragment } from "react";
+import { useCartContext } from "../../contexts/Cart-context/cart-context";
+import "./cart-card.css";
+import { calculateCartItems } from "../../utils/cart-details";
+import { useWishlist } from "../../contexts/wishlist-context/wishlist-context";
+export const CartCard = ({ decideSnackbarFate }) => {
+  const { cartState, cartDispatch } = useCartContext();
+  const { dispatchWish } = useWishlist();
+  return (
+    <Fragment>
+      {cartState.map((item) => {
+        const [discountedPrice, savedAmount] = calculateCartItems(
+          item.price,
+          40
+        );
+        return (
+          <div className="flex flex-direct-col cart-product-container mgn-tp-1 mgn-btm-1 rubik-font border-radius8">
+            <div className="flex">
+              <div className="product-cart-details">
+                <div className="pd-1">{item.name}</div>
+                <div className="flex col-gap-1 jc-center">
+                  <h3>
+                    ₹{(item.qty * discountedPrice).toLocaleString("en-US")}
+                  </h3>
+                  <del>₹{(item.qty * item.price).toLocaleString("en-US")}</del>
+                </div>
+                <div className="green-alert-dark-color text-center">
+                  You saved ₹ {(item.qty * savedAmount).toLocaleString("en-US")}
+                </div>
+                <div className="flex col-gap-1 pd-1">
+                  <div className="grey-0-5-bdr600 pd-0-5--1 border-radius8">
+                    RAM : <span className="font-wt-bold">{item.ram}</span>
+                  </div>
+
+                  <div className="grey-0-5-bdr600 pd-0-5--1 border-radius8 item-cart-qty">
+                    QTY: <span className="font-wt-bold">{item.qty}</span>
+                    <div className="cart-qty-btn-container">
+                      {item.qty < 10 ? (
+                        <button
+                          className="increase-cart-item-qty qty-btn"
+                          onClick={() =>
+                            cartDispatch({
+                              type: "INCREMENT_CART_ITEMS",
+                              payload: item.id,
+                            })
+                          }
+                        >
+                          +
+                        </button>
+                      ) : (
+                        <button
+                          className="increase-cart-item-qty display-btn snck change-btn qty-btn"
+                          onClick={() => {
+                            decideSnackbarFate("INCREASE");
+                            console.log(decideSnackbarFate);
```

Comment:
You can remove the console.log before merging the PR

Reply:
sure, will do that